apps/token/src/routes/rewards/epoch-total-rewards/epoch-total-rewards-table.tsx

```diff
@@ -0,0 +1,212 @@
+import classNames from 'classnames';
+import { useTranslation } from 'react-i18next';
+import { formatNumber } from '@vegaprotocol/react-helpers';
+import { Tooltip, Icon } from '@vegaprotocol/ui-toolkit';
+import { AccountType } from '@vegaprotocol/types';
+import { SubHeading } from '../../../components/heading';
+import type { AggregatedEpochSummary } from './generate-epoch-total-rewards-list';
+
+interface EpochTotalRewardsGridProps {
+  data: AggregatedEpochSummary;
+}
+
+interface ColumnHeaderProps {
+  title: string;
+  tooltipContent?: string;
+  className?: string;
+}
+
+interface RewardItemProps {
+  value: string;
+  dataTestId: string;
+  last?: boolean;
+}
+
+const displayReward = (reward: string) => {
+  if (Number(reward) === 0) {
+    return <span className="text-vega-dark-300">0</span>;
+  }
+
+  if (reward.split('.')[1] && reward.split('.')[1].length > 4) {
+    return (
+      <Tooltip description={formatNumber(reward)}>
+        <button>{formatNumber(Number(reward).toFixed(4))}</button>
```

Comment:
question (non-blocking): What's the button for?

Reply:
I'd grabbed its example use from ui-toolkit where a button was used, but just tried it without and it worked fine. I'll remove it from here and from the stories in ui-toolkit